src/mRides-server/Controllers/UserController.cs

```diff
@@ -47,9 +48,10 @@ public object getUser(int id)
         [HttpPost]
         public object createUser([FromBody]User user1)
         {
-            _userCatalog.createUser(user1);
-            //var r = _context.Users.Include(c => c.RidesAsDriver).Single(u=>u.ID==1);
-            return user1; 
+            //_userCatalog.create(user1);
+            ////var r = _context.Users.Include(c => c.RidesAsDriver).Single(u=>u.ID==1);
+            //return user1;
+            return null; 
```

Comment:
@nim442 Is it normal that `getUser`, `getUserByFacebookId` and `createUser` return null now? Will this affect the calls made from the app side?